test/file_test.rb

```diff
@@ -135,10 +135,11 @@
 
   describe '#retrieve!' do
     before do
-      @file_with_content = RemoteFiles::File.new('identifier', :content => 'content', :content_type => 'content_type')
+      @file_with_content = RemoteFiles::File.new('identifier', :content => 'content', :content_type => 'content_type', :populate_stored_in => nil)
```

Comment:
There should be an additional test with `populate_stored_in` where it checks that `stored_in` does not get set, somewhere [here](https://github.com/zendesk/remote_files/pull/26/files#diff-620c2f7c8c17aca1391ba33ec06310fc68e8470f4a75260a40e6c28a462da446R159)